metadata.json

```diff
@@ -10,44 +10,51 @@
   "dependencies": [
     {
       "name": "puppetlabs-concat",
-      "version_requirement": ">=4.0.0 <7.0.0"
+      "version_requirement": ">=6.0.0 <8.0.0"
     },
     {
       "name": "puppetlabs-inifile",
-      "version_requirement": ">=3.0.0 <5.0.0"
+      "version_requirement": ">=6.0.0 <8.0.0"
```

Comment:
Version 6.0.0 of puppetlabs-inifile [does not exist](https://forge.puppet.com/modules/puppetlabs/inifile).
Please verify all module versions on Puppet Forge before changing them.